### ISSUE TITLE
Multinomial / Multiclass classification implementation

### ISSUE THREAD

**Issue by umangjpatel** [umangjpatel/dnet#2]
**Description of feature request**
Multiclass / Multinomial classification problems not supported.

**Solution proposed**
Implement softmax function

**Additional context**
Refer to docs.fast.ai, PyTorch and Keras documentation for implementation details

**Comment by umangjpatel** [comment]
Multi-class Classification feature implemented in v0.6.2.3
